### DISCUSSION TITLE
Line Width per Entity

### DISCUSSION THREAD

**Discussion by ckglobalroaming** [kalkih/mini-graph-card#1046] [Q&A]
I cant seem to figure out if this is possible? Can anyone share an example if they have fone this please... If not possible would be a great feature request please

Thanks.

**Comment by ildar170975** [comment]
It is not possible currently to define `line_width` per entity.
[There is an open FR.](https://github.com/kalkih/mini-graph-card/issues/555)
Please check for existing issues before opening a new one.
